map_frame: map
point_name: 0
pose:
  position:
    x: -0.3749809265136719
    y: 13.37102699279785
    z: 0
  orientation:
    x: 0
    y: 0
    z: 0.3716626818076588
    w: 0.9283678424803065
